### ISSUE TITLE
Tokenize camel case words.

### ISSUE THREAD

**Issue by ibozhilov** [pyenchant/pyenchant#197]
Hi there,

recently I found out that pyenchant can be used alongside pylint to check the spelling in the docstrings. However, I face a small issue. Sometimes in the documentation or in the examples within the documentation I refer to some classes which are camel case. Those names are flagged as misspelled. Have you consider including in the tokenization process came case words, so that they are checked as separate words and not flagged as misspelled?

**Comment by dmerejkowsky** [comment]
> Have you consider including in the tokenization process came case words, so that they are checked as separate words and not flagged as misspelled?

Yup. I use PyEnchant in my main editor (kakoune) and I would *love* to get this feature.

**Comment by dmerejkowsky** [comment]
Fun fact : I implemented this in Rust using enchant-rs, and know I use that instead of PyEnchant from my main editor

So I no longer need this ...

The source code is here but it maybe harder to use :stuck_out_tongue: 
https://git.sr.ht/~dmerej/skyspell
